htdocs/contact/class/contact.class.php

```diff
@@ -1218,9 +1218,9 @@ public function getCivilityLabel()
 		global $langs;
 		$langs->load("dict");
 
-		$code=(! empty($this->civility_id)?$this->civility:(! empty($this->civilite)?$this->civilite:''));
-		if (empty($code)) return '';
-        return $langs->getLabelFromKey($this->db, "Civility".$code, "c_civility", "code", "label", $code);
+		if (! empty($this->civility)) return $this->civility; // civility is already defined in fetch
```

Comment:
The translation using the code must be used in priority.
The label into table is a fallback when no translation is found.
You fix seems to give the priority to the label into table

Reply:
@eldy i don't understand, civility is defined in fetch function and use the code!

`$this->civility	        = $obj->civility_code?($langs->trans("Civility".$obj->civility_code) != ("Civility".$obj->civility_code) ? $langs->trans("Civility".$obj->civility_code) : $obj->civility_code):'';`

Reply:
@without this fix the civility is not visible in contact card (view mode)

Reply:
@eldy 
look, civility is not visible

![Capture d’écran 2019-08-13 à 07 38 21](https://user-images.githubusercontent.com/454387/62917793-ede16e80-bd9d-11e9-820b-d0e725ffe22f.png)

![Capture d’écran 2019-08-13 à 07 38 35](https://user-images.githubusercontent.com/454387/62917800-f20d8c00-bd9d-11e9-8a9c-5cbae2b2433d.png)